mirascope/core/cohere/call_response.py

```diff
@@ -70,12 +71,12 @@ def content(self) -> str:
         return self.response.text
 
     @property
-    def model(self) -> str | None:
+    def model(self) -> str:
         """Returns the name of the response model.
 
         Cohere does not return model, so we return None
```

Comment:
Update docstring to match? We are returning the model in args, but not the returned call_response model

Reply:
We set `_model` in the specific decorators (like _create) to the model provided by the user, and we use that one if there is no response from the model (like here).

I've updated the docstring.